cPP_DSC.adoc

```diff
@@ -2674,243 +2617,254 @@ FCS_CKM.5.1:: The TSF shall derive cryptographic keys with [*selection*: _key ty
-|{empty}[*selection*: [.underline]#128, 192, 256]# bits 
+|[selection: 128, 192, 256] bits 
 |N/A
 
 |KDF-HASH 
 |Shared secret
 |Hash function from FCS_COP.1/Hash 
-|{empty}[*selection*: [.underline]#128, 192, 256]# bits 
+|[selection: 128, 192, 256] bits 
 |NIST SP 800-56C Rev 2 (Section 4, Option 1)
 
 |KDF-MAC-1S
 |Shared secret, salt, output length, fixed information 
 |Keyed Hash function from FCS_COP.1/KeyedHash
-|{empty}[*selection*: [.underline]#128, 192, 256]# bits 
+|[selection: 128, 192, 256] bits 
 |NIST SP 800-56C Rev 2 (Section 4, Options 2, 3)
 
 |KDF-MAC-2S 
 |Shared secret, salt, IV, output length, fixed information 
 |[MAC Step]
 
-{empty}[*selection*: [.underline]#AES-128-CMAC; AES-192 -CMAC; AES-256 -CMAC; HMAC-SHA-1; HMAC-SHA-256; HMAC-SHA-512]# as the EEF and;
+[selection: AES-128-CMAC, AES-192-CMAC, AES-256-CMAC, HMAC-SHA-1, HMAC-SHA-256, HMAC-SHA-512] as the EEF and;
 
 [KDF Step] 
 
-{empty}[*selection*: [.underline]#KDF-CTR, KDF-FB, KDF-DPI]# using {empty}[*selection*: [.underline]#AES-128-CMAC; AES-192-CMAC; AES-256-CMAC; HMAC-SHA-1; HMAC-SHA-256; HMAC-SHA-512]# as PRF.
+[selection: KDF-CTR, KDF-FB, KDF-DPI] using [selection: AES-128-CMAC, AES-192-CMAC, AES-256-CMAC, HMAC-SHA-1, HMAC-SHA-256, HMAC-SHA-512] as PRF
 
-|{empty}[*selection*: [.underline]#128, 192, 256]# bits 
+|[selection: 128, 192, 256] bits 
 |NIST SP 800-56C Rev 2 (Section 5)
 
-{empty}[*selection*: [.underline]#ISO/IEC 9797-1:2011 (CMAC), NIST SP 800-38B (CMAC), ISO/IEC 18033-3:2010 (AES), ISO/IEC 9797-2:2021 (HMAC), FIPS PUB 198-1 (HMAC), ISO/IEC 10118-3:2018 (SHA), FIPS PUB 180-4 (SHA)]#
+[selection: ISO/IEC 9797-1:2011 (CMAC), NIST SP 800-38B (CMAC), ISO/IEC 18033-3:2010 (AES), FIPS PUB 197 (AES), ISO/IEC 9797-2:2021 (HMAC), FIPS PUB 198-1 (HMAC), ISO/IEC 10118-3:2018 (SHA), FIPS PUB 180-4 (SHA)] 
+
+|KDF-KMAC
+|See NIST 800-108
+|[selection: KMAC128, KMAC256, KMACXOF128, KMACXOF256]
+|[selection: 128, 256] bits
+|[selection: ISO/IEC 9797-2:2021 (Section 9 "MAC Algorithm 4"); NIST SP 800-185 (Section 4 "KMAC")]
 
 |===
 
 _Application Note {counter:remark_count}_:: _The protocol- and application-specific KDFs specified in NIST SP 800-135r1 (e.g., IKE, TLS) and other standards should be covered by SFRs tailored for those protocols. We do expect the cryptographic primitives of application specific KDFs to be validated, e.g., HMAC, SHA. Proper parameters to protocols need to be validated by protocol testing, not cryptographic testing._
 +
-_There are no standards that specify how to derive a key from two keys using XOR (KDF-XOR) or encryption (KDF-ENC)._
+_There are no standards that specify how to derive a key from two keys using XOR (KDF-XOR) or encryption (KDF-ENC). If KDF-XOR is selected, the ST Author should describe this method in the documentation. If KDF-ENC is selected, the ST Author should document the encryption algorithm used from FCS_COP.1/SKC, and which of the inputs is the plaintext and which is the key._
```

Comment:
Since GCM and CCM were removed from FCS_COP.1/SKC, maybe also reference FCS_COP.1/AEAD here?

Reply:
I am not sure how to add that reference properly. @jvdsn please provide a suggestion to the pull request.

Reply:
I added a commit as a suggestion

Reply:
Looks good

Reply:
I went through the catalog adding FSC_COP.1/AEAD where appropriate.